openshift_tools/monitoring/hawk_client.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python2
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+HawkClient uses the openshift_tools.web/rest.py
+
+Example usage:
+
+    from openshift_tools.monitoring.metricmanager import UniqueMetric
+    from openshift_tools.monitoring.hawk_common import HawkConnection
+    from openshift_tools.monitoring.hawk_client import HawkClient
+
+    ml = []
+    # UniqueMetric from openshift_tools.monitoring.metricmanager
+    new_metric = UniqueMetric('example.com','cpu/usage',12.34)
+    ml.append(new_metric)
+    new_metric = UniqueMetric('example.org','cpu/usage',4.2)
+    ml.append(new_metric)
+
+    # No Auth
+    connection = HawkConnection(url='172.17.0.27:8000')
+
+    # Basic Auth
+    connection = HawkConnection(url='172.17.0.27:8000', user='user', passwd='password')
+
+    client = HawkClient(connection)
+    print client.add_metric(ml)
+"""
+# The hawkular client
+from hawkular.metrics import HawkularMetricsClient, MetricType
```

Comment:
@zgalor it's ok to use pip in dev, but we don't commit any code that requires pip.

The hawkular-client RPM will only be available in the rhel container (not centos) since we only do publicly available sources for the centos container.

As a short term fix, since we're disabling the haw_client by default right now anyway, would you mind moving this import down into the constructor _after_ the check to see if the client is disabled?